### PR TITLE
Fix links not working on Docs site

### PR DESCRIPTION
**Pull request by kaankabalak**
## Description

The relative link paths that weren't working have been changed to
direct links to the corresponding Github pages.

## Motivation and Context

The links to `Client grants`, `Configuring wso2`, `Configuring opa`, `Configuring etcd (optional needed only in gateway or federation mode)` in the docs page for [Minio STS Quickstart Guide](https://docs.minio.io/docs/minio-sts-quickstart-guide.html) were not working.

## Regression

No

## How Has This Been Tested?

The commit has been tested by viewing the changed readme and clicking on the changed links.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kannappanr — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

File: docs/sts/README.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ Following are advantages for using temporary credentials:
 - Temporary credentials have a limited lifetime, there is no need to rotate them or explicitly revoke them. Expired temporary credentials cannot be reused.
 
 ## Identity Federation
-[**Client grants**](./client-grants.md) - Let applications request `client_grants` using any well-known third party identity provider such as KeyCloak, WSO2. This is known as the client grants approach to temporary access. Using this approach helps clients keep Minio credentials to be secured. Minio STS client grants supports WSO2, Keycloak.
+[**Client grants**](https://github.com/minio/minio/blob/master/docs/sts/client-grants.md) - Let applications request `client_grants` using any well-known third party identity provider such as KeyCloak, WSO2. This is known as the client grants approach to temporary access. Using this approach helps clients keep Minio credentials to be secured. Minio STS client grants supports WSO2, Keycloak.
```
Comment (deekoder) [inline docs/sts/README.md:14]:
```suggestion
[**Client grants**](https://github.com/minio/minio/blob/master/docs/sts/client-grants.md) - Let applications request `client_grants` using any well-known third party identity provider such as KeyCloak, WSO2. This is known as the client grants approach to temporary access. Using this approach helps clients keep Minio credentials secure. Minio STS client grants support WSO2, Keycloak.
```

Comment (deekoder) [reply docs/sts/README.md:14]: Would we not support and STS client grants. Why only WSO2 and keycloak mentioned here?

**Review by deekoder — CHANGES_REQUESTED** [review]

File: docs/sts/README.md (hunk 1 above)
Comment (harshavardhana) [reply docs/sts/README.md:14]: Just what was tested ..

Comment (deekoder) [reply docs/sts/README.md:14]: can we say Minio STS has been tested with WSO2 and KeyCloak client grants?

Comment (harshavardhana) [reply docs/sts/README.md:14]: We can say in a new PR. This PR is to only fix links.

**Comment by harshavardhana** [comment]
@kaankabalak if you are going to fix more than just links. Change the title accordingly.

File: docs/sts/README.md (hunk 1 above)
Comment (deekoder) [reply docs/sts/README.md:14]: Ok someone will be sending a comprehensive PR for english separately.

**Review by deekoder — APPROVED** [review]

**Comment by kaankabalak** [comment]
@harshavardhana I sent the PR just to correct the links on the README.

**Pull request merged**